.github/workflows/release.yml

```diff
@@ -54,7 +54,8 @@ jobs:
       - uses: actions/checkout@v2
 
       # Handle publishing to our public ECR repo
-      - name: Configure AWS Credentials for SDLC Public ECR
+      - if: github.ref == 'refs/heads/master'
```

Comment:
This check should be moved to the following step, since we want to limit publishing to just master

Reply:
Oops, yeah I had it on the wrong step :face_with_head_bandage: